File: 백준/Bronze/5524. 입실 관리/입실 관리.cs
int num = int.Parse(Console.ReadLine());

for (int i = 0; i < num; i++)
{
    string str = Console.ReadLine();
    foreach (char c in str)
    {
        if(c <= 'Z' && c >= 'A')
        {
            Console.Write((char)(c + 32));
        }
        else
        {
            Console.Write(c);
        }
    }
    Console.WriteLine();
}

File: 백준/Bronze/5524. 입실 관리/입실 관리.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
    public static void main(String[] args)throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int num = Integer.parseInt(br.readLine());
        for(int i = 0; i < num; i++){
            String str = br.readLine();
            for (char c : str.toCharArray()) {
                if(c >= 'A' && c <= 'Z'){
                    System.out.print((char)(c+32));
                }
                else{
                    System.out.print(c);
                }
            }
            System.out.println();
        }
        
    }
}